yt_dlp/extractor/nitter.py

```diff
@@ -265,6 +265,26 @@ class NitterIE(InfoExtractor):
                 'repost_count': int,
                 'comment_count': int,
             }
+        }, {  # test for OpenGraph patch #7575
+            'url': f'https://nitter.net/LocalBateman/status/1678455464038735895#m',
```

Comment:
```suggestion
            'url': f'https://{current_instance}/LocalBateman/status/1678455464038735895#m',
```

Reply:
Hey, I had hardcoded the domain since not all domains worked for me when testing locally, must have forgotten to change that back. Should I create a new pull request?

Reply:
A new PR to update the domains/instances? Only if you want to. 

Otherwise we can just apply these suggestions and the instances can be updated at some point in the future